typings/camelize.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module 'camelize' {
```

Comment:
It'd be nice to submit these to DefinitelyTyped at some point, though it's fine that they're here for now.